cdc/entry/schema_storage.go

```diff
@@ -89,19 +107,21 @@ func NewSchemaStorage(
 			return nil, errors.Trace(err)
 		}
 	}
-	return &schemaStorageImpl{
+	result := &schemaStorage{
```

Comment:
It is better to simply return it rather than assigning it to result and then returning the result.